index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Portfolio - ZC</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="./icons/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <!-- Toolbar -->
+    <div class="toolbar">
+        <p id="toolbartext">Welcome</p>
+        <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
```

Comment:
- [ ] Kindly😃 try to put the logo inside a link This is for three reasons: convention, semantics, and user experience(UX)

> 😊⤵️

| Logo |
| --- |
|    ![image](https://user-images.githubusercontent.com/83360636/219173589-5ab08956-66de-40be-832f-cab11f26c966.png)  |